File: app/src/main/java/com/hqq/core/app/dialog/FoldFullBottomSheet.java
package com.hqq.core.app.dialog;

import androidx.recyclerview.widget.LinearLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import com.hqq.core.ui.BaseBottomDialog;
import com.hqq.core.utils.ResourcesUtils;
import com.hqq.core.app.R;
import com.hqq.core.app.adapter.StringAdapter;

/**
 * @Author : huangqiqiang
 * @Package : com.hqq.coreapp.dialog
 * @FileName :   FullBottomSheet
 * @Date : 2019/4/29 0029  下午 2:06
 * @Email : qiqiang213@gmail.com
 * @Descrive :
 */
public class FoldFullBottomSheet extends BaseBottomDialog {
    StringAdapter mStringAdapter = new StringAdapter();

    @Override
    public int setViewId() {
        return R.layout.dialog_bottom_sheet;

    }

    @Override
    public void initView() {
        RecyclerView recyclerView = mRootView.findViewById(R.id.rc_list);
        recyclerView.setLayoutManager(new LinearLayoutManager(getContext()));
        for (int i = 0; i < 20; i++) {
            mStringAdapter.addData("    " + i);
        }
        recyclerView.setAdapter(mStringAdapter);
    }

    @Override
    public int setHeight() {
        return (int) ResourcesUtils.getDimen(getContext(), R.dimen.x750);
    }

    @Override
    protected int getTransparentBottomSheetStyle() {
        return R.style.TransparentBottomSheetStyle_behavior;
    }
}

File: app/src/main/java/com/easy/example/dialog/FoldFullBottomSheet.kt
package com.easy.example.dialog

import androidx.recyclerview.widget.LinearLayoutManager
import androidx.recyclerview.widget.RecyclerView
import com.easy.core.ui.dialog.BaseBottomDialog
import com.easy.example.R
import com.easy.example.adapter.StringAdapter

/**
 * @Author : huangqiqiang
 * @Package : com.easy.coreapp.dialog
 * @FileName :   FullBottomSheet
 * @Date : 2019/4/29 0029  下午 2:06
 * @Email : qiqiang213@gmail.com
 * @Describe :
 */
class FoldFullBottomSheet : BaseBottomDialog() {
    var mStringAdapter = StringAdapter()


    override fun getDialogWeight(): Int {
        TODO("Not yet implemented")
    }

    override fun getLayoutViewId(): Int {
      return R.layout.dialog_bottom_sheet
    }
    override fun initView() {
        val recyclerView: RecyclerView = rootView!!.findViewById(R.id.rc_list)
        recyclerView.layoutManager = LinearLayoutManager(context)
        for (i in 0..19) {
            mStringAdapter.add("    $i")
        }
        recyclerView.adapter = mStringAdapter
    }

    override val transparentBottomSheetStyle: Int
        get() = R.style.TransparentBottomSheetStyle_behavior


}